services/vision/mlvision/ml_model.go

```diff
@@ -210,8 +217,35 @@ func registerMLModelVisionService(
 	return vision.NewService(name, r, nil, classifierFunc, detectorFunc, segmenter3DFunc)
 }
 
+func getLabelsFromFile(labelPath string) []string {
```

Comment:
Validate will already check if the labelPath exists, which is why this is a private function